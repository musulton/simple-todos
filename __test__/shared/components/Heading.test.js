import {render, screen} from "@testing-library/react-native";

import Heading from "../../../src/shared/components/Heading";

describe("Heading", () => {
    test("should render correctly", () => {
        render(<Heading />)
        
        expect(screen.toJSON()).toMatchSnapshot()
    })
})
