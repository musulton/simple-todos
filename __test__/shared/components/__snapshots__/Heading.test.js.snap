// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading should render correctly 1`] = `
<View
  style={
    {
      "marginTop": 40,
    }
  }
>
  <Text
    style={
      {
        "color": "rgba(175, 47, 47, 0.25)",
        "fontSize": 72,
        "fontWeight": "100",
        "textAlign": "center",
      }
    }
  >
    todos
  </Text>
</View>
`;
